Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -5705,7 +5705,7 @@ final class WorkspaceTests: XCTestCase {
                     throw StringError("unexpected url \(request.url)")
                 }
 
-                try fileSystem.writeFileContents(
+                try fs.writeFileContents(
```

Comment:
I think it is better to use the FS coming from the handler as it potentially could be different (eg VFS)